skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -0,0 +1,43 @@
+package org.jetbrains.skiko
+
+// Clicked mouse buttons bitmask.
+object SkikoMouseButtons {
```

Comment:
If we have renamed MouseEvent to PointerEvent, we probably should rename it too:
```
object SkikoPointerButtons {
    const val NONE = 0
    const val PRIMARY = 1 shl 0
    const val SECONDARY = 1 shl 1
    const val TERTIARY = 1 shl 2
    const val BACK = 1 shl 3
    const val FORWARD = 1 shl 4
}
```

or rename `buttons` to `mouseButtons` in SkikoPointerEvent.